src/reducer/reducerFunction.jsx

```diff
@@ -0,0 +1,80 @@
+const reducerFunction = (state, action) => {
+  switch (action.type) {
+    case "PROD_TO_LOAD": {
+      return { ...state, products: action.payload };
+    }
+    case "SORT_BY_PRICE": {
+      return { ...state, sortPrice: action.payload };
+    }
+    case "SORT_BY_RATING": {
+      return { ...state, sortRating: action.payload };
+    }
+    case "SUNGLASSES": {
+      return {
+        ...state,
+        sortCategory: {
+          ...state.sortCategory,
+          bySunglasses: !state.sortCategory.bySunglasses,
```

Comment:
you can use variable for SORT_BY_PRICE and for other cases

Reply:
It didnt clicked for me....will do the change after finishing projects.